languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -21171,6 +21171,64 @@ USA
                 <example>“I see it makes you laugh, Tom?”</example>
             </rule>
         </rulegroup>
+        <rulegroup id="DO_MAKE_PRP_VBG" name="make it working -> make it work" default="temp_off">
+            <rule>
+                <antipattern>
+                    <token postag="PRP$" />
+                    <token>living</token>
+                </antipattern>
+                <antipattern>
+                    <token regexp="yes">and|could</token>
+                    <token min="0">'ve</token>
+                    <token inflected="yes">make</token>
+                </antipattern>
+                <antipattern>
+                    <token>going</token>
+                    <token>forward</token>
+                </antipattern>
+                <antipattern>
+                    <token>being</token>
+                    <token min="0" postag="RB" />
+                    <token>behind</token>
+                </antipattern>
+                <pattern>
+                    <token inflected="yes">make</token>
+                    <token postag="PRP">
+                        <exception>everyone</exception>
+                    </token>
+                    <marker>
+                        <token postag="VBG">
+                            <exception regexp="yes">annoying|appealing|binding|boring|challenging|confusing|daunting|deserving|doing|embarassing|embarrassing|exciting|frustrating|interesting|pleasing|relaxing|rewarding|scaling|sickening|willing</exception>
```

Comment:
How about replacing this entire word collection with `&adjectives_with_ing;`?

Reply:
Replaced.